### ISSUE TITLE
Walk style/Animation menu

### ISSUE THREAD

**Issue by Cozivillain** [sneakyevil/SleepingDogs-ModMenu#22]
hey sneaky, Your menu is awesome and i think adding walkstyle swaps and an animation menu would make it even more awesome, Thanks for your hard work.

**Comment by sneakyevil** [comment]
No, clue if this will be added since how hard it would be to somehow modify specific action nodes of player behaviour to playback different action nodes for movement.
